pipeline/cron-pipeline.groovy

```diff
@@ -0,0 +1,97 @@
+/*
+    Pipeline script for triggering scheduled cron job.
+*/
+
+def nodeName = "centos-7"
+def sharedLib
+def buildArtifacts
+def release
+
+
+node(nodeName) {
+    try {
+        timeout(unit: "MINUTES", time: 30) {
+            stage('Install prereq') {
+                if (env.WORKSPACE) { sh script: "sudo rm -rf * .venv" }
+                checkout(
+                    scm: [
+                        $class: 'GitSCM',
+                        branches: [[name: 'refs/remotes/origin/cron_grv']],
```

Comment:
Could we change the branch to `origin/master`

Reply:
done